cmd/explorer/main.go

```diff
@@ -569,6 +569,8 @@ func main() {
 
 			router.HandleFunc("/monitoring/{module}", handlers.Monitoring).Methods("GET", "OPTIONS")
 
+			router.HandleFunc("/lukso/supply", handlers.LuksoSupply).Methods("GET")
```

Comment:
Im wondering if we can name this endpoint in more RESTful way, like `/stats/supply`, what do you think?